src/ClassDesignChallenge/UniversityInfo.java

```diff
@@ -0,0 +1,26 @@
+package ClassDesignChallenge;
+
+import java.time.LocalDate;
+
+public class UniversityInfo {
+    public static void main(String[] args) {
+
+        Professors professor1 = new Professors("John", "Connor", 'M', LocalDate.of(1989, 12, 12), 33, 1891212050432L, "Canada, 8th Avenue, Apartment 12");
+        Professors professor2 = new Professors("May", "Russel", 'F', LocalDate.of(1991, 10, 5), 30, 2911005050431L, "New Orleans, Spring Avenue, Apartment 57");
```

Comment:
Do you think you can create a professor but this time with Builder Design Pattern?
We didn't discuss Design Patterns in Java, but it's something that can be done, considering your skills.
We didn't approach constructors either, yet you used them.

Reply:
Constructor are in the Java OOP Concepts.pdf file. For some reason I can't remember well if we've discussed it or not however I had a look at the entire pdf from scratch and it was easier to understand. I think the last topic was the discussion of static methods so I just had a read onwards. Will take a look today at the Builder Design Patter, sounds extremely nerdy so I will probably love it.